src/App.tsx

```diff
@@ -82,19 +84,43 @@ const router = createBrowserRouter(
 )
 
 export const App = (): React.ReactElement => {
-  const client = useContext(KeycloakContext)
-  const setClient = useCredentialStore((state) => state.setClient)
+  const client = useContext(KeycloakContext);
+  const setClient = useCredentialStore((state) => state.setClient);
 
   useEffect(() => {
-    setClient(client)
-  }, [])
+    setClient(client);
+  }, [client, setClient]);
+
+  const removeAllCookies = () => {
+    const allCookies = Cookies.get();
+    Object.keys(allCookies).forEach(cookieName => {
+      Cookies.remove(cookieName, { path: '/' });
+    });
+  };
```

Comment:
You can set up Prettier with the following configuration by saving this script as`.prettierrc.json` in the root directory
```
{
  "semi": false,
  "trailingComma": "all",
  "singleQuote": true,
  "printWidth": 80,
  "tabWidth": 2
}
```
It will help you format the file before saving it.